aws/resource_aws_cognito_user_pool.go

```diff
@@ -1154,6 +1142,22 @@ func resourceAwsCognitoUserPoolUpdate(d *schema.ResourceData, meta interface{})
 		}
 	}
 
+	if d.HasChange("schema") {
+		oldSchema, newSchema := d.GetChange("schema")
+		if oldSchema.(*schema.Set).Difference(newSchema.(*schema.Set)).Len() == 0 {
```

Comment:
it might also be helpful to document this behavior for users. in the resource docs there's already a note I believe, so you could append to it stating schema items can be added but not removed or updated

Reply:
Good call!